frontend-react/src/utils/OrganizationUtils.ts

```diff
@@ -96,9 +57,69 @@ export const groupToOrg = (group: string | undefined): string => {
     }
 };
 
-export function useOrgName(): string {
-    const org = useResource(OrganizationResource.detail(), {
-        name: groupToOrg(getStoredOrg()),
+/* Delivers the array of organization names, or the first, parsed from Okta
+ * groups. */
+const getRSOrgs = (
+    token: AccessToken | undefined,
+    onlyFirst: boolean = false,
+    onlyType?: RSOrgType
```

Comment:
I want to refactor this so `onlyType` -> `type`, and it defaults to `"all"`, and remove `onlyFirst`.